### PR TITLE
merging to main.

### PR DESCRIPTION
**Pull request by deusalexmachina**

### CODE REVIEW

File: restaurant/views.py [hunk 1]
```diff
@@ -186,40 +185,7 @@ def get_restaurants_list(request, page):
...
-        user,
-    )
-
-    parameter_dict = {
-        "restaurant_list": json.dumps(restaurant_list, cls=DjangoJSONEncoder),
-        "page": page,
-        "keyword": json.dumps({"keyword": keyword}),
-    }
-    return render(request, "browse.html", parameter_dict)
+    return render(request, "browse.html")
```
Comment (Chuanqi927) [inline restaurant/views.py:188]: Are we not using the previous function for landing page?

**Review by Chuanqi927 — COMMENTED** [review]
one question regarding get_landing_page in views

File: restaurant/views.py (hunk 1 above)
Comment (deusalexmachina) [reply restaurant/views.py:188]: yes

**Review by Chuanqi927 — APPROVED** [review]
Approve

**Pull request merged**